packages/frontapp/src/app/pages/HistoryOfClaimPage.tsx

```diff
@@ -27,9 +27,8 @@ export const List = styled.div`
   max-height: 650px;
   overflow-y: auto;
```

Comment:
I think it works better if you remove this `overflow-y`
Because there are users who don't use mouse wheel for scrolling, they won't be able to scroll without scrollbar